Classes/Emogrifier.php

```diff
@@ -422,6 +419,17 @@ protected function process(\DOMDocument $xmlDocument)
             $this->fillStyleAttributesWithMergedStyles();
         }
 
+        if ($this->shouldMapCssToHtml) {
```

Comment:
Please let's move this to a new, well-named private method so this method doesn't grow any longer, and the good name will make understanding the method easier.